project/src/components/app/app.tsx

```diff
@@ -1,5 +1,11 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import Main from '../main/main';
+
+ type AppProps = {
+   countOffer: number;
+ }
```

Comment:
```suggestion
type AppProps = {
  countOffer: number;
}
```
лишние отступы нужно убрать